src/scripts/components/foundations/Title.js

```diff
@@ -1,16 +1,5 @@
 import styled from 'styled-components'
 
 export default styled.header`
-  background-color: #fff;
-  height: 50px;
-  padding: 20px;
-  display: flex;
-  align-items: center;
-  color: #818181;
-  font-weight: 600;
-  margin-bottom: 15px;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  white-space: nowrap;
-  width: 100%;
+  color: white;
```

Comment:
Can we use one of our defined color variables here?

Reply:
It can. This is not defined yet